### ISSUE TITLE
[BUG] Examples method list_examples not returning all examples from a given playbook

### ISSUE THREAD

**Issue by frrleone** [GoogleCloudPlatform/dfcx-scrapi#221]
When using the Examples class, the list_examples method is not returning all the examples for the provided playbook. Some playbooks that have 20+ examples are returning only a handful, or even none at all.

## Expected Behavior

The expected behaviour is that all the examples present in a given playbook are returned by the list_examples method.

## Current Behavior

Current behaviour is that not all examples (and sometimes none of the examples) in a given playbook are returned by the list_examples method.

## Steps to Reproduce

1. Have a DFCX agent containing a playbook which itself contains multiple examples
2. Call list_examples with that playbook's ID
3. The returned list of examples is not the complete list of examples present in the web console

Code I used to test that it isn't returning all of the examples:

dfcx_pb = Playbooks(creds=credentials, agent_id = agent_id)
dfcx_ex = Examples(creds=credentials, agent_id = agent_id)
playbooks_list = dfcx_pb.list_playbooks()

for playbook in playbooks_list:
  print(str(len(dfcx_ex.list_examples(playbook.name))) + " examples in playbook " + playbook.display_name)

(it's unclear to me if this is always the case, it was the case for all 20+ playbooks in the agent I am working in)

## Context (Environment)

The issue has prevented me from being able to export the data for all the examples present in a given playbook. I'm looking to ensure they are all configured correctly with the necessary input parameters for this client's use case.

**Comment by kmaphoenix** [comment]
Ok I figured this one out after some digging.
It's a bug in the core library so I have filed a bug with the Engineering team directly.  
I'll keep this issue open for public tracking until the internal issue is resolved and then I'll report back here.  

# TLDR
Try `list_examples(playbook_id, language_code="undetected")` to find your missing examples.

# Details
The Examples service in Playbooks / Agent Builder attempts to auto-detect the `language_code` that the example is written in.
A byproduct of this is that if the language code is unclear or if the auto detect fails for some reason, the example is stored with `language_code=undetected` like this:
![image](https://github.com/user-attachments/assets/fe74a158-d399-4403-8bac-b78d979cd01b)

By default, when you call `list_examples()` it uses `language_code=en`.  
So it's finding all the Examples that it marked as `en` but failing to return any examples marked as `undetected`.  
This would happen the same for other lang codes like `es`, `fr`, etc.

# Workaround
A workaround for now is to call `list_examples()` twice:
- 1x for your lang_code
- 1x for `undetected` as the lang_code

**Comment by kmaphoenix** [comment]
Filed `361766559` for internal tracking.

**Comment by kmaphoenix** [comment]
Fix for this is now applied in prod.
Closing this issue as no change is need on the SCRAPI side of things since this was a product side bug.
